quick-protocol.html

```diff
@@ -0,0 +1,1186 @@
+
+                      <dl rel="skos:hasTopConcept">
+                        <dt about="#ResourceServer" property="skos:prefLabel" rel="skos:broadMatch" resource="spec:RespondingAgent" typeof="skos:Concept"><dfn id="ResourceServer">Resource Server</dfn></dt>
+                        <dd about="#ResourceServer" property="skos:definition">A <em>resource server</em> builds on an HTTP <cite>server</cite> [<cite><a class="bibref" href="#bib-rfc9110">RFC9110</a></cite>] by defining media types, HTTP header fields, and the behaviour of resources, as identified by link relations.</dd>
+                        <dt about="#SubscriptionClient" property="skos:prefLabel" rel="skos:broadMatch" resource="spec:Consumer" typeof="skos:Concept"><dfn id="SubscriptionClient">Subscription Client</dfn></dt>
+                        <dd about="#SubscriptionClient" property="skos:definition">A <em>Subscription Client</em> that builds on HTTP <cite>client</cite> [<cite><a class="bibref" href="#bib-rfc9110">RFC9110</a></cite>] and [<cite><a class="bibref" href="#bib-fetch">FETCH</a></cite>] by defining behaviour in terms of fetching across the platform.</dd>
+                        <dt about="#NotificationSender" property="skos:prefLabel" rel="skos:broadMatch" resource="spec:RespondingAgent" typeof="skos:Concept"><dfn id="NotificationSender">Notification Sender</dfn></dt>
+                        <dd about="#NotificationSender" property="skos:definition" rel="skos:broadMatch" resource="spec:ProducerOfContent">A <em>Notification Sender</em> is the sender of a notification acting in conformance to the product class of a notification channel type.</dd>
+                        <dt about="#NotificationReceiver" property="skos:prefLabel" rel="skos:broadMatch" resource="spec:RespondingAgent" typeof="skos:Concept"><dfn id="NotificationReceiver">Notification Receiver</dfn></dt>
+                        <dd about="#NotificationReceiver" property="skos:definition" rel="skos:broadMatch" resource="spec:Consumer">A <em>Notification Receiver</em> is the recipient of a notification acting in conformance to the product class of a notification channel type.</dd>
+                       </dl>
+                    </div>
+                  </section>
+
+                  <section id="interoperability" inlist="" rel="schema:hasPart" resource="#interoperability" typeof="skos:ConceptScheme">
+                    <h4 property="schema:name skos:prefLabel">Interoperability</h4>
+                    <div datatype="rdf:HTML" property="schema:description">
+                      <p property="skos:definition">Interoperability occurs between <a href="#subscription-client-resource-server-interoperability" rel="skos:hasTopConcept">Subscription Client–Resource Server</a> or <a href="#notification-sender-notification-receiver-interoperability" rel="skos:hasTopConcept">Notification Sender–Notification Receiver</a>, as defined by the <a href="" rel="foaf:page">Solid Quick Notifications Protocol</a>.</p>
```

Comment:
```suggestion
                      <p property="skos:definition">Interoperability occurs between <a href="#subscription-client-resource-server-interoperability" rel="skos:hasTopConcept">Subscription Client↔Resource Server</a> or <a href="#notification-sender-notification-receiver-interoperability" rel="skos:hasTopConcept">Notification Sender↔Notification Receiver</a>, as defined by the <a href="" rel="foaf:page">Solid Quick Notifications Protocol</a>.</p>
```

Reply:
Need to check with Sarven on the arrows.